main/opcodes/create-terminate-context.zkasm

```diff
@@ -996,17 +1008,20 @@ opSENDALL2:
     $ => A                  :MLOAD(storageAddr)
     0 => D
     $ => SR                 :SSTORE
+
+opSENDALLend:
     ; if origin ctx is 0, end tx
     $ => A                  :MLOAD(originCTX)
-    $                       :EQ,JMPC(handleGas)
+    A                       :JMPZ(handleGas)
+
     ; decrease CTX
     $ => B                  :MLOAD(gasRefund)
     $ => C                  :MLOAD(isCreate)
     1 => D
```

Comment:
WE could do this with one step less like this I guess:
....
$ => B                  :MLOAD(gasRefund)
1 => D
$ => C                  :MLOAD(isCreate), JMPZ(opSENDALLendContinue)
$ => D                  :MLOAD(createContractAddress)

opSENDALLendContinue:
....

Reply:
added